### PR TITLE
New demos for color axis 

### PR DESCRIPTION
**Pull request by kamil-musialowski**
Added new API demos for `colorAxis.stops` and `series.colorKey`.

### CODE REVIEW

File: samples/highcharts/coloraxis/coloraxis-stops/demo.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                    [0.5, '#5c81b5'],
+                    [0.6, '#5470b3'],
+                    [0.9, '#2843b8'],
+                    [1, '#0f11a6']
+                ]
+            }
+        });
+    });
+
+    document.getElementById('darkThemeBtn').addEventListener('click', function () {
```
Comment (raf18seb) [inline samples/highcharts/coloraxis/coloraxis-stops/demo.js:37]: Same as above

**Review by raf18seb — CHANGES_REQUESTED** [review]
Nice demos! I have a few comments, see below

**Review by raf18seb — APPROVED** [review]
Thanks!

**Review by TorsteinHonsi — APPROVED** [review]
Thanks, nice demos!

**Pull request merged**